### PR TITLE
added day to logger and a little refactored

### PR DESCRIPTION
**Pull request by paledrewnem**
Short Description: 

Fixes:
- added current day to logger

### CODE REVIEW

**Comment by douglascamata** [comment]
need to fix conflicts.

**Comment by paledrewnem** [comment]
Ready to review!

**Comment by douglascamata** [comment]
@paledrewnem did you test?

File: pokemon_spawn_locations.json [hunk 1]
```diff
@@ -1,3 +0,0 @@
-{
```
Comment (TheSavior) [inline pokemon_spawn_locations.json]: Why are you deleting this file?

**Comment by paledrewnem** [comment]
@douglascamata  OK, I tested it and it works

@TheSavior sorry, my bad. Thanks for being watchful :)

**Comment by surfaace** [comment]
:-1: No need for day to console. it was there before and we decided to remove it

**Pull request closed without merging**